package.json

```diff
@@ -84,7 +84,7 @@
   },
   "devDependencies": {
     "coveralls": "2.11.9",
-    "eslint-config-fxa": "2.1.0",
+    "eslint-plugin-fxa": "git://github.com/mozilla/eslint-plugin-fxa.git#master",
```

Comment:
@divyabiyani long story short, we do not "shrinkwrap" dependencies in the fxa-content-server, can you change this from `master` to the GIT SHA of the `eslint-plugin-fxa` you want to use here